qa/L0_backend_python/argument_validation/models/argument_validation/1/model.py

```diff
@@ -145,14 +145,14 @@ def test_tensor_args(self):
             pb_utils.Tensor("OUTPUT0", None)
 
         # Test None as dlpack capsule
-        with self.assertRaises(TypeError) as e:
+        with self.assertRaises(pb_utils.TritonModelException) as e:
             pb_utils.Tensor.from_dlpack("OUTPUT0", None)
 
-        # Test empty string as model name (from_dlpack)
-        with self.assertRaises(TypeError) as e:
+        # Test empty string as tensor name (from_dlpack)
+        with self.assertRaises(pb_utils.TritonModelException) as e:
```

Comment:
do we need to test with a proper dlpack (not None) but invalid name?

Reply:
I'll check that